vnext/Chakra/ChakraExecutor.cpp

```diff
@@ -209,20 +213,20 @@ JsErrorCode ChakraExecutor::RedirectConsoleToDebugger(JsValueRef debuggerConsole
 
   const char *script =
       "function patchConsoleObject$$1(global, console, debugConsole) {\n"
-      "var obj = {};\n"
-      "for (var fn in console) {\n"
-      "if (typeof console[fn] === \"function\") {\n"
-      "(function(name) {\n"
-      "obj[name] = function(...args) {\n"
-      "console[name](...args);\n"
-      "if (name in debugConsole && typeof debugConsole[name] === \"function\") {\n"
-      "debugConsole[name](...args);\n"
-      "}\n"
-      "}\n"
-      "})(fn);\n"
-      "}\n"
-      "}\n"
-      "global.console = obj;\n"
+        "var obj = {};\n"
+        "for (var fn in console) {\n"
+          "if (typeof console[fn] === \"function\") {\n"
+            "(function(name) {\n"
+              "obj[name] = function(...args) {\n"
+              "console[name](...args);\n"
+              "if (name in debugConsole && typeof debugConsole[name] === \"function\") {\n"
+                "debugConsole[name](...args);\n"
+              "}\n"
+            "}\n"
+            "})(fn);\n"
+          "}\n"
+        "}\n"
+        "global.console = obj;\n"
```

Comment:
nit, could get rid of the \n and spaces since this isn't designed to be used/seen by humans

Reply:
also you'll need to run yarn format which will remove the leading spaces before the string starts too